cmd/chartsvc/handler.go

```diff
@@ -444,3 +445,75 @@ func newChartVersionListResponse(c *models.Chart) apiListResponse {
 
 	return cvl
 }
+
+func handleRedirect(w http.ResponseWriter, req *http.Request) {
```

Comment:
perhaps this should be named `getChartVersionPackage`? Looks like most of the handlers in this file start with `get` or `list`. Another option is `redirectToChartVersionPackage`.

Reply:
I used `redirectToChartVersionPackage` because it's not getting the version package... it's more getting a redirect to the package.